opp_gui/src/widgets/tool_path_planner_widget.cpp

```diff
@@ -307,11 +308,7 @@ void ToolPathPlannerWidget::saveModel()
   TouchPointMap verification_points = verification_point_editor_->getPoints();
   if (touch_points.size() < MIN_TOUCH_POINTS || verification_points.size() < MIN_VERIFICATION_POINTS)
   {
-    QMessageBox::warning(this,
-                         "Invalid Model Definition",
-                         ("Ensure at least " + std::to_string(MIN_TOUCH_POINTS) + " touch points and " +
-                          std::to_string(MIN_VERIFICATION_POINTS) + " verification points have been defined")
-                             .c_str());
+    QWarningBox("Ensure at least 3 touch points and 3 verification points have been defined");
```

Comment:
Basic geometry indicates that these will not change. However, we no longer require using touch off points for localization, so the variable is set to 0 even though it takes 3. If there is one touch of point there should be at least three. If there are none, that is ok because we assume they are not in use.